gst/nnstreamer/tensor_query/tensor_query_common.h

```diff
@@ -74,33 +76,58 @@ typedef struct
   };
 } TensorQueryCommandData;
 
-/**
- * @brief generate unique id.
- * @return unique id if OK, 0 if error
- */
-extern uint64_t
-nnstreamer_query_request_id (const char *ip, uint32_t port, int is_client);
-
 /**
  * @brief connect to the specified address.
  * @return 0 if OK, negative value if error
  */
 extern int
-nnstreamer_query_connect (uint64_t id, const char *ip, uint32_t port, uint32_t timeout_ms);
+nnstreamer_query_connect (void * connection, const char *ip, uint32_t port, uint32_t timeout_ms);
 
 /**
  * @brief send command to connected device.
  * @return 0 if OK, negative value if error
  */
 extern int
-nnstreamer_query_send (uint64_t id, TensorQueryCommandData *data, uint32_t timeout_ms);
+nnstreamer_query_send (void * connection, TensorQueryCommandData *data, uint32_t timeout_ms);
 
 /**
  * @brief receive command from connected device.
  * @return 0 if OK, negative value if error
  */
 extern int
-nnstreamer_query_receive (uint64_t id, TensorQueryCommandData *data, uint32_t timeout_ms);
+nnstreamer_query_receive (void * connection, TensorQueryCommandData *data, uint32_t timeout_ms);
+
+/**
+ * @brief close connection with corresponding id.
+ * @return 0 if OK, negative value if error
+ */
+extern int
+nnstreamer_query_close (void * connection);
+
+/* server */
+extern void *
+nnstreamer_query_server_accept (void *server_data);
```

Comment:
Need more explanation for external interfaces

Reply:
Fixed!